audio/audio_driver.c

```diff
@@ -1291,12 +1401,128 @@ static void audio_driver_mixer_play_stream_internal(unsigned i, unsigned type)
       audio_mixer_streams[i].state   = (enum audio_mixer_state)type;
 }
 
+static void audio_driver_load_menu_bgm_callback(void *task_data, void *user_data, const char *error)
```

Comment:
Should we need a `HAVE_MENU` check around this? Don't need the audio driver menu sounds if there's no menu.